modules/@apostrophecms/modal/ui/apos/composables/AposFocus.js

```diff
@@ -22,8 +25,9 @@ export function useAposFocus() {
   // `cycleElementsToFocus` listeners relies on this dynamic list which has the advantage of
   // taking new or less elements to focus, after an update has happened inside a modal,
   // like an XHR call to get the pieces list in the AposDocsManager modal, for instance.
-  function cycleElementsToFocus(e) {
-    if (!elementsToFocus.value.length) {
+  function cycleElementsToFocus(e, elements) {
+    const elems = elements || elementsToFocus.value;
+    if (!elems.length) {
```

Comment:
The power of the composable, modals want to pass elements because they come from the store. But in `AposButtonSplit` it simply uses the provided `elementsToFocus` from this composable.